llvm/lib/Target/SBF/SBFISelLowering.cpp

```diff
@@ -466,9 +462,17 @@ SDValue SBFTargetLowering::LowerFormalArguments(
       EVT PtrVT = DAG.getTargetLoweringInfo().getPointerTy(DAG.getDataLayout());
       EVT LocVT = VA.getLocVT();
 
+      unsigned Offset;
+      if (Subtarget->getHasDynamicFrames()) {
+        // In SBFv2, the arguments are stored on the start of the frame.
+        Offset = VA.getLocMemOffset() + 8;
```

Comment:
are the 8 bytes added to offset the stack pointer? Maybe get the size of the pointer using getSizeInBytes() on the corresponding pointer value type? Would be good to avoid using a hard-coded constant if possible.

Reply:
The offset is added to the frame pointer. The offset zero corresponds to top (lowest address) of the callee function and the offset 8 is the first addressable frame location from the executing function. This is why I had to add eight to everything.